cmd/configure.go

```diff
@@ -104,7 +109,8 @@ var configureBaseURLCmd = &cobra.Command{
 		}
 
 		if len(args) == 0 {
-			return cmd.Help()
+			fmt.Println(viper.GetString("override_base_url"))
+			return nil
```

Comment:
Maybe we want to configure this to be slightly more user-friendly 
```suggestion
			baseURL := viper.GetString("override_base_url")
			message := fmt.Printf("Base URL: %s", baseURL)
			if baseURL == "" {
			    message := "No Base URL set"
			}
			fmt.Println(message)
			return nil
```